menpo/io/input/base.py

```diff
@@ -68,7 +68,7 @@ def same_name_video(path, frame_number):
     return {p.suffix[1:].upper(): p for p in landmark_file_paths(pattern)}
 
 
-def import_image(filepath, landmark_resolver=same_name, normalise=True):
+def import_image(filepath, landmark_resolver=same_name, normalize=True):
```

Comment:
Perfect, but since this will break compatibility I would prefer than we deprecate this for a version. So maintain the `normalise` spelling as well as then new `normalize` and raise a warning if `normalise` is used.